CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 0.16.0 (15 July 2021)
+* Documentation updates (#800, #818, #839, #842)
+* Fixed data alert repr in subscription item (#821)
+* Added support for Data Quality Warning (#836)
+* Updated request factory to add name attribute (#843)
```

Comment:
Can you add datasource to be more specific to what changed in request factory? Or maybe something like "Added support for renaming datasources"

Reply:
Yes, that is much better and reflective of what that PR was. Thanks! Fixed it in the next commit.